cfgov/unprocessed/css/molecules/image-text-50-50.less

```diff
@@ -82,7 +82,6 @@
 */
 
 .m-image-text-50-50 {
-    padding-bottom: unit( @grid_gutter-width * 2 / @base-font-size-px, em );
```

Comment:
An option could be doing 
```
&+& {
  padding-top: unit( @grid_gutter-width * 2 / @base-font-size-px, em );
}
```
so that it only gets added when there's more than one and it won't affect the flow following them.


Side thought, why aren't we using margin?